src/napari/_qt/layer_controls/widgets/qt_opacity_blending_controls.py

```diff
@@ -0,0 +1,126 @@
+        Label for the blending combobox widget.
+    opacity_slider : superqt.QLabeledDoubleSlider
+        Slider controlling opacity of the layer.
+    opacity_label : napari._qt.layer_controls.widgets.qt_widget_controls_base.QtWrappedLabel
+        Label for the opacity slider widget.
+    """
+
+    def __init__(self, parent: QWidget, layer: Layer) -> None:
+        super().__init__(parent, layer)
+        # Setup layer
+        self._layer.events.blending.connect(self._on_blending_change)
+        self._layer.events.opacity.connect(self._on_opacity_change)
+
+        # Setup widgets
+        sld = QLabeledDoubleSlider(Qt.Orientation.Horizontal, parent=parent)
+        sld.setFocusPolicy(Qt.FocusPolicy.NoFocus)
+        sld.setMinimum(0)
+        sld.setMaximum(1)
+        sld.setSingleStep(0.01)
+        self.opacity_slider = sld
+        connect_setattr(
+            self.opacity_slider.valueChanged, self._layer, 'opacity'
+        )
+        self.opacity_label = QtWrappedLabel(trans._('opacity:'))
+        self._on_opacity_change()
+
+        blend_combobox = QComboBox(parent)
+        for index, (data, text) in enumerate(BLENDING_TRANSLATIONS.items()):
+            data = data.value
+            blend_combobox.addItem(text, data)
+            if data == self._layer.blending:
+                blend_combobox.setCurrentIndex(index)
+
+        blend_combobox.currentTextChanged.connect(self.change_blending)
+        self.blend_combobox = blend_combobox
+        self.blend_label = QtWrappedLabel(trans._('blending:'))
+
+        # opaque and minimum blending do not support changing alpha
+        self.opacity_slider.setEnabled(
+            self._layer.blending not in NO_OPACITY_BLENDING_MODES
+        )
+        self.opacity_label.setEnabled(
+            self._layer.blending not in NO_OPACITY_BLENDING_MODES
+        )
+
+    def change_blending(self, text: str) -> None:
+        """Change blending mode on the layer model.
+
+        Parameters
+        ----------
+        text : str
+            Name of blending mode, eg: 'translucent', 'additive', 'opaque'.
+        """
+        self._layer.blending = self.blend_combobox.currentData()
+        # opaque and minimum blending do not support changing alpha
+        self.opacity_slider.setEnabled(
+            self._layer.blending not in NO_OPACITY_BLENDING_MODES
+        )
+        self.opacity_label.setEnabled(
+            self._layer.blending not in NO_OPACITY_BLENDING_MODES
+        )
+
+        blending_tooltip = ''
+        if self._layer.blending == str(Blending.MINIMUM):
+            blending_tooltip = trans._(
+                '`minimum` blending mode works best with inverted colormaps with a white background.',
+            )
+        self.blend_combobox.setToolTip(blending_tooltip)
+        self._layer.help = blending_tooltip
+
+    def _on_opacity_change(self) -> None:
+        """
+        Receive layer model opacity change event and update opacity slider.
```

Comment:
```python 
str(Blending.MINIMUM) == Blending.MINIMUM
Out[6]: True
```
```suggestion
        if self._layer.blending == Blending.MINIMUM:
```

Reply:
This suggestion was meant for a few lines above I assume!

Reply:
yes